### PR TITLE
Polish translation improvement from Crowdin made by:

### PR DESCRIPTION
**Pull request by magul**
* Daga (https://crowdin.com/profile/dagaaaaa)
* Monika Czerniawska (https://crowdin.com/profile/wrozda)
* Zuzzka (https://crowdin.com/profile/Zuzzka)

CC: @Zuzzka, @wrozda, @dagaaaaa

### CODE REVIEW

File: pl/django_forms/README.md [hunk 1]
```diff
@@ -376,7 +376,7 @@ Jeżeli potrzebujesz więcej informacji o formularzach Django, zajrzyj do dokume
 
 Super, że możemy tworzyć posty klikając w link! Jest niestety jeden problem - aktualnie każdy, kto odwiedzi Twoją stronę, może stworzyć nowy post. Zróbmy tak, aby przycisk był widoczny tylko dla Ciebie i nikogo innego.
 
-W `blog/templates/blog/base.html` znajdź `page-header` `div` i tag <0>a</0>, który umieściliśmy tam wcześniej. Powinien wyglądać tak:
+Otwórzmy `blog/templates/blog/base.html` w edytorze kodu i znajdźmy nasz `page-header` `div` i zalinkuj tag, który umieściliśmy tam wcześniej. Powinien wyglądać tak:
```
Comment (w1stler) [inline pl/django_forms/README.md]: First sentence is not correct.

File: pl/django_forms/README.md [hunk 2]
```diff
@@ -398,7 +398,7 @@ Ten `{% if %}` spowoduje, że link zostanie wysłany do przeglądarki tylko wted
 
 Pamiętasz ikonkę edycji, która przed chwilą dodałyśmy do naszej strony szczegółów? Chcemy też dodać tę samą zmianę tutaj tak, by inne osoby nie były w stanie edytować istniejących postów.
 
-Otwórz `blog/templates/blog/post_detail.html` i znajdź poniższą linijkę:
+Otwórzmy `blog/templates/blog/post_detail.html` w edytorze kodu i dodaj poniższą linijkę:
```
Comment (w1stler) [inline pl/django_forms/README.md]: Otwórzmy (...) dodaj - also not correct.

File: pl/django_forms/README.md [hunk 3]
```diff
@@ -444,6 +444,6 @@ Dobrze byłoby wiedzieć, że nasza witryna nadal działa na PythonAnywhere, pra
 
 (Pamiętaj by podmienić `<your-pythonanywhere-username>` na twoją właściwą nazwę użytkownika PythonAnywhere, bez nawiasów ostrokątnych).
 
-* Na koniec, przejdźmy do podstrony [Web tab](https://www.pythonanywhere.com/web_app_setup/) i wciśnijmy **Reload** (ang. odśwież).
+* Nareszcie przeskakujemy do do ["Web" page](https://www.pythonanywhere.com/web_app_setup/) (użyj przycisku menu w prawym górnym rogu konsoli) i naciśnij **Reaload**. Odśwież swoj blog https://yourname.pythonanywhere.com, aby zobaczyć zmiany.
```
Comment (w1stler) [inline pl/django_forms/README.md]: `Reload`

File: pl/django_models/README.md [hunk 4]
```diff
@@ -95,7 +95,7 @@ Zauważysz, że nowy katalog `blog` został utworzony i zawiera kilka plików. K
     └── requirements.txt
     
 
-Po stworzeniu aplikacji, musimy dać znać Django, że powinien jej używać. Posługujemy się w tym celu plikiem `mysite/settings.py`. Musimy odnaleźć nagłówek `INSTALLED_APPS` i dodać wiersz o treści `'blog',` tuż przed nawiasem zamykającym `]`. Czyli efekt końcowy powinien wyglądać tak:
+Po stworzeniu aplikacji, musimy dać znać Django, że powinien jej używać. Robimy to w pliku `mysite/settings.py` -- otwórz go w swoim edytorze kodu. Musimy odnaleźć nagłówek `INSTALLED_APPS` i dodać wiersz o treści `'blog',` tuż przed nawiasem zamykającym `]`. Czyli efekt końcowy powinien wyglądać tak:
```
Comment (w1stler) [inline pl/django_models/README.md]: double dash

File: pl/django_start_project/README.md [hunk 5]
```diff
@@ -80,7 +80,7 @@ W pliku `settings.py` znajdź linijkę, w której jest napisane `TIME_ZONE` i zm
 TIME_ZONE = 'Europe/Warsaw'
 ```
 
-Kod języka składa się z języka, np. ` en ` dla języka angielskiego lub ` de ` dla języka niemieckiego i kodu kraju, np. ` de ` dla Niemiec lub ` ch ` dla Szwajcarii. Jeżeli angielski nie jest Twoim językiem ojczystym, możesz dodać tę wartość, aby domyślne przyciski i powiadomienia od Django były w Twoim języku. Tak więc przycisk "Anuluj" zostałby przetłumaczony na język, który tu zdefiniowałeś. [Django jest wyposażone w mnóstwo przygotowanych tłumaczeń](https://docs.djangoproject.com/en/2.0/ref/settings/#language-code).
+Kod języka składa się z języka, np. ` en ` dla języka angielskiego lub ` de ` dla języka niemieckiego i kodu kraju, np. ` de ` dla Niemiec lub ` ch ` dla Szwajcarii. Jeżeli angielski nie jest Twoim językiem ojczystym, możesz dodać tę wartość, aby domyślne przyciski i powiadomienia od Django będą w Twoim języku. Tak więc przycisk "Anuluj" zostałby przetłumaczony na język, który tu zdefiniowałeś. [Django jest wyposażone w mnóstwo przygotowanych tłumaczeń](https://docs.djangoproject.com/en/2.0/ref/settings/#language-code).
```
Comment (w1stler) [inline pl/django_start_project/README.md]: zdefiniowałeś - ditto

File: pl/django_templates/README.md [hunk 6]
```diff
@@ -97,7 +97,7 @@ Dobrze byłoby sprawdzić, czy twoja strona wciąż będzie działała w publicz
     [...]
     
 
-* Na koniec przejdź do podstrony [Web tab](https://www.pythonanywhere.com/web_app_setup/) i naciśnij **Reload**. Twoja aktualizacja powinna być widoczna! Jeśli posty na blogu w witrynie PythonAnywhere nie pasują do postów znajdujących się na blogu hostowane na lokalnym serwerze, jest to OK. Bazy danych na komputerze lokalnym i Python Anywhere nie synchronizują się z resztą plików.
+* Na koniec przeskocz do strony [Web](https://www.pythonanywhere.com/web_app_setup/) i naciśnij **Reload**. (Aby dotrzeć do innych stron PythonAnywhere przez konsolę, użyj przycisku menu w prawym górnym rogu.) Twoja aktualizacja powinna być już pod adresem https://yourname.pythonanywhere.com -- sprawdź to w swojej przeglądarce! Jeśli posty na blogu w witrynie PythonAnywhere nie pasują do postów znajdujących się na blogu hostowane na lokalnym serwerze, jest to OK. Bazy danych na komputerze lokalnym i Python Anywhere nie synchronizują się z resztą plików.
```
Comment (w1stler) [inline pl/django_templates/README.md]: Redundant parentheses in first sentence?

Comment (w1stler) [reply pl/django_templates/README.md]: Double dash

File: pl/django_urls/README.md [hunk 7]
```diff
@@ -98,6 +98,6 @@ Jeśli teraz spróbujesz odwiedzić stronę http://127.0.0.1:8000/, zobaczysz ko
 
 ![Błąd](images/error1.png)
 
-Twoja konsola pokazuje błąd, ale nie martw się - w rzeczywistości jest to całkiem użyteczne: mówi Ci, że** brak atrybutu 'post_list'**. To jest nazwa widoku (*view*), którą Django próbuje znaleźć i użyć, ale jeszcze go nie utworzyłyśmy. Na tym etapie Twój `/admin/ ` również nie będzie działać. Nie martw się, zajmiemy się tym.
+Twoja konsola pokazuje błąd, ale nie martw się - w rzeczywistości jest to całkiem użyteczne: mówi Ci, że** brak atrybutu 'post_list'**. To jest nazwa widoku (*view*), którą Django próbuje znaleźć i użyć, ale jeszcze go nie utworzyłyśmy. Na tym etapie Twój `/admin/ ` również nie będzie działać. Nie martw się, zajmiemy się tym. Jeżeli pojawiła Ci się jakaś inna wiadomość o innym błędzie, spróbuj zrestartować swój serwer. Aby to zrobić, przejdź do wiersza polecenia w którym uruchomiony jest serwer strony, zatrzymaj serwer wciskając Ctrl+C (klawisze Control i C równocześnie), a następnie uruchom go jeszcze raz poleceniem `python manage.py runserver`.
```
Comment (w1stler) [inline pl/django_urls/README.md]: inna, innym - repetition

File: pl/chromebook_setup/instructions.md [hunk 8]
```diff
@@ -1,19 +1,34 @@
...
+3. Kliknij przycisk *New Server*
+4. Kliknij przycisk "Terminal" (po lewej stronie okna)
+
+Teraz powinieneś zobaczyć interfejs z bocznym paskiem i przyciski po lewej stronie. Kliknij przycisk "Terminal", aby otworzyć okno terminala ze znakiem zachęty takim jak ten:
+
+{% filename %}Terminal{% endfilename %}
+
+    $
+    
+
```
[the remark this answers is not in the document]
Comment (magul) [reply pl/chromebook_setup/instructions.md]: That's how Crowdin redner this file. I have checked that even before openning this pull request and haven't figure out how to solve it.

File: pl/deploy/README.md [hunk 9]
```diff
@@ -207,15 +205,15 @@ Jeśli widzisz błąd podczas uruchamiania skryptu `pa_autoconfigure_django.py`,
...
 
 Są tam też dostępne [generalne porady odnośnie debugowania na stronie pomocy PythonAnywhere](http://help.pythonanywhere.com/pages/DebuggingImportError).
 
 I pamiętaj, Twój mentor jest tutaj, by Ci pomóc!
 
 # Sprawdź swoją stronę!
 
-Domyślna strona Twojej witryny powinna brzmieć "It worked!", tak jak na Twoim lokalnym komputerze. Spróbuj dodać `/admin/` na koniec swojego adresu URL - powinnaś się przenieść do panelu admina. Zaloguj się przy użyciu nazwy użytkownika i hasła, a zobaczysz, że możesz dodawać nowe posty na serwerze.
+Domyślna strona Twojej witryny powinna brzmieć "It worked!", tak jak na Twoim lokalnym komputerze. Spróbuj dodać `/admin/` na koniec swojego adresu URL - powinnaś się przenieść do panelu admina. Zaloguj się za pomocą nazwy użytkownika i hasła. Zobaczysz, że możesz dodać nowy Post na serwerze -- pamiętaj, posty z Twojej lokalnej, testowej bazy danych nie będą wysyłane na Twojego produkcyjnego bloga.
```
[the remark this answers is not in the document]
Comment (magul) [reply pl/deploy/README.md]: Actually double dashes were introduced in English version (see eg: https://github.com/DjangoGirls/tutorial/commit/60677487ef88cb84e107d0b9fc8ef7d69144e875).

Don't know what they do but they are there.

**Comment by magul** [comment]
@w1stler I just pushed fixes by @wrozda.

File: pl/chromebook_setup/instructions.md (hunk 8 above)
[the remark this answers is not in the document]
Comment (w1stler) [reply pl/chromebook_setup/instructions.md]: Let's leave it then.

**Pull request merged**